### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by dblessing**
Added a standard code of conduct adapted from http://contributor-covenant.org/version/1/1/0/

### CODE REVIEW

**Comment by martinb3** [comment]
Maybe we could cite the Chef Community guidelines as well?

**Comment by dblessing** [comment]
As long as they don't conflict. Maybe it would be better to copy theirs so they are in agreement?

**Comment by zuazo** [comment]
Chef Community Guidelines here: https://docs.chef.io/community_guidelines.html

Your proposal sounds good anyway. Short and clear :+1:

**Review by damacus — APPROVED** [review]

**Review by zuazo — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Chef Brigade Contributor Code of Conduct
```
Comment (shortdudey123) [inline CODE_OF_CONDUCT.md]: Name change

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+As contributors and maintainers of projects within Chef Brigade, we pledge to respect all people
```
Comment (shortdudey123) [inline CODE_OF_CONDUCT.md]: Here too

**Review by shortdudey123 — CHANGES_REQUESTED** [review]

**Comment by damacus** [comment]
@shortdudey123 That should fix those issues.

**Review by shortdudey123 — APPROVED** [review]

**Comment by josephholsten** [comment]
We voted to used a modified version of the chef code of conduct, so this is no longer correct.

**Pull request closed without merging**